src/com/raghav/javabasics/day4/SplitStringIntoSubstrings.java

```diff
@@ -0,0 +1,41 @@
+package com.raghav.javabasics.day4;
+// Given a string and an index, write a program to split the string into two substrings based on the index.
+
+public class SplitStringIntoSubstrings {
+    public static void main(String[] args) {
+        SplitStringIntoSubstrings result = new SplitStringIntoSubstrings();
+        result.splitString("",21);
```

Comment:
Output:
String is Empty/Null
String is Empty/Null
First substring: a   Second substring: bcdefg
Enter Valid Index
First substring: abcde   Second substring: fg
Enter Valid Index
First substring: abcdefg   Second substring: 
First substring: abc      Second substring: defg
First substring: ab$^%.   Second substring: .cdefg
Enter Valid Index